Hazel/src/Hazel/Scene/Components.h

```diff
@@ -45,11 +46,22 @@ namespace Hazel {
 	struct SpriteRendererComponent
 	{
 		glm::vec4 Color{ 1.0f, 1.0f, 1.0f, 1.0f };
+		Ref<Texture2D> Texture = nullptr;
```

Comment:
Why is a "nullptr" used here?!

"NULL" also works fine.

Reply:
`nullptr` is fine in my opinion. 🙂